x-pack/plugins/observability_solution/apm/server/lib/helpers/create_es_client/create_assets_es_client/create_assets_es_clients.ts

```diff
@@ -45,30 +50,45 @@ export async function createEntitiesESClient({
   request: KibanaRequest;
   esClient: ElasticsearchClient;
 }) {
+  function search<TDocument = unknown, TSearchRequest extends ESSearchRequest = ESSearchRequest>(
+    indexName: string,
+    operationName: string,
+    searchRequest: TSearchRequest
+  ): Promise<InferSearchResponseOf<TDocument, TSearchRequest>> {
+    const controller = new AbortController();
+
+    const promise = withApmSpan(operationName, () => {
+      return cancelEsRequestOnAbort(
+        esClient.search(
+          { ...searchRequest, index: [indexName] },
+          {
+            signal: controller.signal,
+            meta: true,
+          }
+        ) as unknown as Promise<{
+          body: InferSearchResponseOf<TDocument, TSearchRequest>;
+        }>,
+        request,
+        controller
+      );
+    });
+
+    return unwrapEsResponse(promise);
+  }
+
   return {
     search<TDocument = unknown, TSearchRequest extends ESSearchRequest = ESSearchRequest>(
```

Comment:
nit: I'm thinking it would be better  to rename it `searchLatest` following the `searchHistory`